projects/sw-background-sync-queue/src/index.js

```diff
@@ -11,7 +11,9 @@
  limitations under the License.
  */
 
-import initialize from './lib/initialization';
+import requestManager from './lib/requestManager';
 import pushIntoQueue from './lib/enqueue';
 
+var initialize = requestManager.initialize;
```

Comment:
This is another one of those places where you might want to get consistent with the use of `let`/`const` vs `var`